docs/reference/yaml.md

```diff
@@ -239,7 +239,60 @@ The meanings and formats of `limits` and `requests` may vary depending on whethe
 
 See docs for [Docker Swarm](https://docs.docker.com/config/containers/resource_constraints/) or for [Kubernetes](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#how-pods-with-r    esource-limits-are-run).
 
-### YAML - environment variable substitution
+## Configuration
+The configuration section allows you to define additional configuration that is global to the entire stack, currently this mostly impacts function build time options.
+
+### Templates
```

Comment:
The `Function` section of this document has all the sub-sections prefixed with the parent: `Function: Memory/CPU Limits`
For consistency, these sub-sections should be `Configuration: Templates` etc

On the other hand, the prefix may be too much, and the other sub-sections should have that prefix removed. The ToC on the right of the page already formats them in a way to identify that they are sub-sections of the `Function` or `Configuration`.

Either way, I just think it should be consistent

Reply:
I think this section is a little different because the `configuration` key is a static value.  The `Function` prefix is there because those yaml keys are under a user supplied name instead of a static value.

Reply:
I see your point, and that makes a lot of sense